<ui version="4.0" >
 <class>managePPCclientClass</class>
 <widget class="QMainWindow" name="managePPCclientClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>managePPCclient</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QTableWidget" name="ipTableWidget" >
          <property name="midLineWidth" >
           <number>0</number>
          </property>
          <property name="rowCount" >
           <number>2</number>
          </property>
          <property name="columnCount" >
           <number>3</number>
          </property>
          <row/>
          <row/>
          <column>
           <property name="text" >
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Owner</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePPCStatusButton" >
          <property name="text" >
           <string>Update PPC Status</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>PPC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipLineEdit" >
            <property name="text" >
             <string>172.16.20.</string>
            </property>
            <property name="readOnly" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="applyPPCButton" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Apply PPC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="giveBackPPCButton" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Give Back PPC</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QPushButton" name="quitButton" >
          <property name="text" >
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>261</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel" >
        <property name="text" >
         <string>Disconnected to server</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
